src/components/Header/Header.jsx

```diff
@@ -20,7 +19,6 @@ const Header = () => {
                         <img src={airbnbLogo} className={styles.logoAirbnb} alt='The Logo of AirBnB' />
                     </a>
                 </div>
-				<ToggleButtonsStaysExperiences />
```

Comment:
![image](https://github.com/user-attachments/assets/ef433035-c0a1-4b6d-8ca1-6c2afe3e9ff7)
    In the product page also if you click on the search bar it collapses and gives the detail search bar and it also shows the stays and experiences selection. So I feel this component should stay on the header.

Reply:
Moved the component to the Header